code/game/antagonist/station/revolutionary.dm

```diff
@@ -7,7 +7,7 @@ var/datum/antagonist/revolutionary/revs
 	bantype = "revolutionary"
 	feedback_tag = "rev_objective"
 	antag_indicator = "rev_head"
-	welcome_text = "Down with the capitalists! Down with the Bourgeoise!"
+	welcome_text = "You are a Head Revolutionary! It's your job to whip the crew into a frenzy against Command and Central! <b>Use the uplink disguised as a station-bounced radio in your backpack to help start the revolution!</b>"
```

Comment:
I disagree with the wording of that.
The purpose of the headrev isnt to start a deathmatch, it is to work with the head loyalists to provide a interesting story.